en/banners/vpn-holidays.ftl

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+banner-vpn-holidays-title = { -brand-name-mozilla-vpn }
+
+# strong tag is for visual formatting only.
+banner-vpn-holidays-take-20-percent-off = This holiday season <strong>take 20% off the first year</strong>
```

Comment:
Insert a comma after `<strong>`.

Reply:
Do you mean before? It won't look right to have a comma bolded before the other boldened words?

Reply:
Yes, I meant before the tag. I happened to review the Relay PR today and it uses the same string: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/165.

Reply:
Oh sorry! I don't know why i read "after" instead of "before". Must have been my morning brain :) I'll fix that up shortly.

Reply:
So, looking into where the string is, this is to be in a banner and have a `<br>` between "season" and `<strong>`. There should be a separation between `This holiday season` and the rest of the string. So I don't think a comma is really required? The relay string shows it to be a sentence within a promo rather than a headline. I forgot to add the `<br>` in the code when I was working on this last week, but happy to add it in 

![image](https://github.com/mozilla-l10n/www-l10n/assets/42309026/0588e9b6-1a5d-41fa-9514-904cdda485b5)

Reply:
Got it. Let's add a <br> so localizers have a sense how the sentence is structured. So this will be the only change and we are all good.